packages/router/utils.ts

```diff
@@ -1533,11 +1533,21 @@ export const redirectDocument: RedirectFunction = (url, init) => {
   return response;
 };
 
+export interface ErrorResponse {
```

Comment:
Can we keep this as a `type` to avoid changing the semantics to `interface`?

Reply:
Sure, we can, but from my point of view it exposes the public API, so it is exactly an interface semantically, or am I missing something? Maybe you want to avoid interface merging?

Reply:
yeah, we prefer `type` over `interface` unless we need the ability for end users to augment the `interface`, which we shouldn't in this case

Reply:
Done